src/main/java/com/softserve/mapper/LessonsListInRoomScheduleMapper.java

```diff
@@ -0,0 +1,38 @@
+package com.softserve.mapper;
+
+import com.softserve.dto.LessonsInRoomScheduleDTO;
+import com.softserve.dto.LessonsListInRoomScheduleDTO;
+import com.softserve.entity.Lesson;
+import com.softserve.entity.Schedule;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.List;
+
+@Mapper(componentModel = "spring")
+public abstract class LessonsListInRoomScheduleMapper {
+
+    protected GroupMapper groupMapper;
+
+    public LessonsListInRoomScheduleMapper() {}
+
+    @Autowired
+    public void setGroupMapper(GroupMapper groupMapper) {
```

Comment:
You can just set uses = IdentifierMapper.class in your @Mapper annotation

Reply:
https://stackoverflow.com/questions/60523230/how-can-i-use-another-mapping-from-different-class-in-mapstruct